src/dex/fluid-dex/fluid-dex.ts

```diff
@@ -303,6 +293,7 @@ export class FluidDex extends SimpleExchange implements IDex<FluidDexData> {
         }, ${side}:`,
         e,
       );
+      throw e;
```

Comment:
Returning null should be enough, no need in throwing the error

Reply:
done